project1.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="project1.css" type="text/css">
+  <link href="https://font.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+  integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <title>Setup and Mobile Version Skeleton</title>
+</head>
+
+<body>
+  <header>
+
+<section>
+
+<div class="flex-container1">
+
+<div class="snapshoot">
+  <a><img src="Snapshoot Portfolio.png" alt=" "></a>
+</div>
+
+<div class="flex-container12">
+<div class="one1"><h1>Multi-Post Stories</h1>
+</div>
+
+<div class="two2">
+
+<div class="canopy"> CANOPY </div>
+<div class="counter1"> <a><img src="Counter.jpg" alt=" "></a></div>
+<div class="back"> Back End Dev </div>
+<div class="counter2"> <a><img src="Counter.jpg" alt=" "></a></div>
+<div class="year"> 2015 </div>
+
+</div>
+
+<div class="three3"><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+
+<div>
+
+  <ul>
+    <li>
+      <button class="button1">html</button>
+      <button class="button2">css</button>
+      <button class="button3">javascript</button>
+    </li>
+  </ul>
+
+</div>
+
+<div><button class="button4">See Project</button></div>
+
+</div>
+</div>
+
+<div class="flex-container">
+<div class="about"><h1>About Myself</h1></div>
+
+  <div class="dev"><p>I'm a software developer! I can help you build a product, feature or website
+    Look through some of my work and experience! If you like what you see and have
+    a project you need coded, don't hesitate to contact me.</p></div>
+
+   <div class="one">LET'S CONNECT </div>
+
+<div>
+  <ul>
+     <li><a ><img src="github.jpg" alt=" " class="github"></a></li>
+    <li><a ></a><img src="Linkedinicon.jpg" alt=" " class="linkedin"></a></li>
+      <li><a ><img src="angellist.jpg" alt=" " class="angellist"></a></li>
+     <li><a ><img src="twitter.jpg" alt=" " class="twitter"></a></li>
+      <li><a ><img src="medium.jpg" alt=" " class="medium"></a></li>
+    </ul>
```

Comment:
- [ ] OPTIONAL
Your code is **not properly indented** and therefore your code formatting is inconsistent. Kindly ensure that your code has consistent **spacing** for indentation. The recommended spacing is **2 spaces**. You can install and configure a code formatter plugins such as [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) or [Beautify](https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify) in your code editor for formatting your code to ensure consistent and well-indented code.